css/doc-explorer.css

```diff
@@ -114,16 +114,32 @@
 }
 
 .graphiql-container .doc-category-title {
-  border-bottom: 1px solid #e0e0e0;
-  color: #777;
+  color: #fff;
   cursor: default;
+  font-family: 'Roboto Mono';
   font-size: 14px;
-  font-variant: small-caps;
   font-weight: bold;
   letter-spacing: 1px;
-  margin: 0 -15px 10px 0;
+  margin: 0 -15px 10px 10px;
   padding: 10px 0;
   user-select: none;
+  position: relative;
+  width: fit-content;
+  text-transform: capitalize;
```

Comment:
If this is supposed to turn query fields to look like type names... that's not going to work. We need to use the actual type names from the schema, because not all types created by users start with e.g. a capital letter. We don't require that at the moment.

Reply:
Hehe, definitely not :) This is only to style the title and fit that purple underline underneath it 
<img width="142" alt="screen shot 2019-02-27 at 8 51 20 am" src="https://user-images.githubusercontent.com/1865664/53507505-e424b280-3a6c-11e9-8d0b-748be3996700.png">